Purge "will" from Admin Guide (#662)
uyuni-project/uyuni-docs: modules/administration/pages/maintenance-windows.adoc

[NOTE]
====
When you assign a new maintenance schedule to a client, it is possible that the client might already have some restricted actions scheduled, and that these might now conflict with the new maintenance schedule.

> If this occurs, the {webui} displays an error and you can not assign the schedule to the client.

Reviewer: ```suggestion
If this occurs, the {webui} displays an error and you cannot assign the schedule to the client.
```
